pom.xml

```diff
@@ -76,7 +76,7 @@
     <spotbugs.threshold>${findbugs.threshold}</spotbugs.threshold>
     <!-- Defines a SpotBugs effort. Use "Max" to maximize the scan depth -->
     <spotbugs.effort>${findbugs.effort}</spotbugs.effort>
-    
+
```

Comment:
revert ws changes please

Reply:
:man_facepalming: I definitely staged only the bottom initially, but then I think I did a mistake before the commit. Done